packages/eslint-plugin-orbit-components/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../config/eslintTS.js",
+  "rules": {
+    "@typescript-eslint/prefer-readonly-parameter-types": "off",
+    "@typescript-eslint/explicit-function-return-type": "off"
+  }
+}
```

Comment:
Put these rules in `overrides` in root `.eslintrc.js` instead, and delete this file. The `eslintTS.js` file is incidental and we'll probably merge it with `.eslintrc.js` later.